src/api.yaml

```diff
@@ -0,0 +1,1285 @@
+openapi: 3.0.1
+info:
+  title: Session Open Group Server
+  description: >
+    API documentation for Session open groups.  This is the API that [Session](https://getsession.org)
+    and related tools use to interface with open groups.
+  contact:
+    name: The Oxen Project
+    email: team@oxen.io
+    url: https://getsession.org
+  license:
+    name: GPL v3.0
+    url: https://www.gnu.org/licenses/gpl-3.0.en.html
+  version: "3.0"
+externalDocs:
+  description: Find out more about the Oxen project
+  url: http://oxen.io
+tags:
+- name: Rooms
+  description: Access to a server's rooms (AKA open groups) and room management
+- name: Messages
+  description: Message-related endpoints for accessing, submitting, deleting, etc.
+- name: Batch
+  description: Endpoint for submitting a batch of requests at once.
+- name: Users
+  description: Management of users and user-related info.
+security:
+  - pubkey: []
+    nonce: []
+    timestamp: []
+    signature: []
+paths:
+  /rooms:
+    get:
+      tags: [Rooms]
+      summary: "Returns a list of available rooms on the server."
+      description: >
+        Rooms to which the user does not have access (e.g. because they are banned) are not included.
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Room"
+  /room/{roomToken}:
+    get:
```

Comment:
ditto with fields filter here

Reply:
Even less reason to bother here; the Room response isn't that large.